### ISSUE TITLE
Option to keep internal game fps and speed, on high refresh rate display

### ISSUE THREAD

**Issue by RaduNastase** [skyline-emu/skyline#2317]
Suggestion:
An option to keep the framerate throttled to normal game's 60 (or 30 depending on the game) even if the screen is set to high refresh rates (ex. 160 hz)
Now, even with the Framerate Throttle option on default (meaning throttling is active), when i activate the maximum refresh rate, the games lose the Throttling and go fast, or just mess up the fps completely.

The refresh rate of the display should idealy (and if possible) be sepparate from the game's actual fps, like in all other emulators. Peculiar thing is, that a few games (Rolling Gunner, Ninja Gaiden Sigma2) actually "keep" the Throttle active even at max display refresh rate.  
Basically they play at 60 fps, but with the benefits of the ultra fast touch response of 160 hz, which is what the whole point is.
